modules/upgrade/pages/server-major-upgrade-uyuni.adoc

```diff
@@ -0,0 +1,88 @@
+[server-major-upgrade]
+= Server - Major Upgrade
+
+From time to time, {productname} Server upgrades are labeled as major.
+This is especially the case, if a version upgrade of openSUSE Leap is needed. openSUSE Leap is the underlying base operating system (OS).
+For background information about upgrading openSUSE Leap, see https://doc.opensuse.org/documentation/leap/startup/single-html/book-opensuse-startup/index.html#sec-update-zypper.
+
+Run the [command]``/usr/lib/susemanager/bin/server-migrator.sh`` script to upgrade the base OS and {productname} Server.
+
+[NOTE]
+====
+If the [command]``server-migrator.sh`` script is not already installed on your {productname} Server system, update the [package]``susemanager`` package:
++
+----
+zypper ref
+zypper up susemanager
+----
+====
+
+The [command]``server-migrator.sh`` script will migrate {productname} Server to the latest version.
+This implies upgrading the underlying operating system to version 15.2.
+
+// During migration the services need to be shut down and after successful migration the server needs to be rebooted manually.
+
+[IMPORTANT]
+====
+Because you cannot fix any issues during migration, make sure you have a backup before starting the migration.
+If you are running {productname} Server on a virtual machine, it is recommended to create a snapshot before performing the migration.
+====
+
+.Procedure:
+
+The script runs as [systemitem]``root`` and performs the following actions.
+
+
+. Stop the spacewalk services:
```

Comment:
I don't think this section is explaining the procedure.

The procedure is what you wrote above.

And why do we need to explain the script line by line? Would be enough if you list what it does, but without adding the lines, FMPOV.

What value adds having the script content?

Reply:
I thought it would help the user if user wants to write an own version of the script. But if you think it will cause confusion, I can remove it.
BTW, in those scripts we should always use long options for readability.

Reply:
The user does not need to write their own version. They could mess the migration.

What do you mean with long options?

Reply:
Why was this conversation merged, the PR merged, and the script content still there?